documentation/modules/configuring/con-config-examples.adoc

```diff
@@ -53,12 +46,4 @@ examples
 <6> `Kafka` custom resource configuration for a deployment of Kafka. Includes example configuration for an ephemeral or persistent single or multi-node deployment.
 <7> `Kafka` custom resource with a deployment configuration for Cruise Control. Includes `KafkaRebalance` custom resources to generate optimizations proposals from Cruise Control, with example configurations to use the default or user optimization goals.
 <8> `KafkaConnect` and `KafkaConnector` custom resource configuration for a deployment of Kafka Connect. Includes example configuration for a single or multi-node deployment.
-<9> `KafkaBridge` custom resource configuration for a deployment of Kafka Bridge.
-
-== CPU and memory resource limits and requests
-
-By default, the Strimzi Cluster Operator does not specify CPU and memory resource requests and limits for its deployed operands.
-Ensuring an adequate allocation of resources is crucial for maintaining stability and achieving optimal performance in Kafka.
-The ideal resource allocation depends on your specific requirements and use cases.
-
-It is recommended to configure CPU and memory resources for each container by link:{BookURLConfiguring}#con-common-configuration-resources-reference[setting appropriate requests and limits^].
```

Comment:
Is this covered somewhere else? It seems to be useful.

Reply:
Yes, we have it somewhere else